fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -88,6 +89,12 @@ def self.available_options
                                        description: "Pass in xcpretty additional command line arguments (e.g. '--test --no-color' or '--tap --no-utf'), requires xcpretty_output to be specified also",
                                        type: String,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :skip_update,
+                                       env_name: "FL_SPM_SKIP_UPDATE",
+                                       description: "Skip updating dependencies from their remote during a resolution",
+                                       optional: true,
+                                       type: Boolean,
+                                       default_value: false),
```

Comment:
Since this value is a boolean, I think we can remove the optionality and have it default to false as we're already doing 😃 (there's no point in explicitly passing `nil` in this case)

```suggestion
                                       type: Boolean,
                                       default_value: false),
```